libnetwork/portallocator/portallocator_unix.go

```diff
@@ -9,5 +9,5 @@ const (
 	// defaultPortRangeEnd indicates the last port in port range
 	// consistent with default /proc/sys/net/ipv4/ip_local_port_range
 	// upper bound on linux
-	defaultPortRangeEnd = 60999
+	defaultPortRangeEnd = 65535
```

Comment:
Let me comment on this line; this may have been related to https://github.com/moby/moby/pull/40055#issuecomment-1145398915 as well (in case we need to un-revert this change in range)

/cc @neersighted @corhere @evol262 

(and myself, for if this causes havoc in our products 🙈)